addons/web/static/src/views/fields/text/text_field.js

```diff
@@ -18,6 +22,43 @@ export class TextField extends Component {
                 this.resize();
             }
         });
+        onMounted(this.onMounted);
+        onWillUnmount(this.onWillUnmount);
```

Comment:
I'm surprised that this would be needed ?

Reply:
I'm pretty sure it is. 
As I understand it, Owl expect you to register theses kind of stuff yourself when you need them.